src/openforms/ui/static/ui/scss/components/skiplink/_skiplink.scss

```diff
@@ -1,7 +1,16 @@
 .skiplink {
   position: absolute;
-  top: -100vh;
-  left: -100vh;
+  padding: 6px;
+  background-color: var(--of-skiplink-bg, var(--of-button-primary-bg));
+  color: var(--of-skiplink-fg, var(--of-button-primary-fg));
+  border-bottom-right-radius: 6px;
+  border-right: 1px solid var(--of-skiplink-color-border, var(--of-button-primary-color-border));
+  border-bottom: 1px solid var(--of-skiplink-color-border, var(--of-button-primary-color-border));
```

Comment:
this breaks when a different theme is used for OF that doesn't have our "3D" buttons.

Must this be styled as a button, or could it also be a plain link/anchor? In that case, slapping the `utrecht-link` class on the element should be sufficient.

Reply:
discussed - we'll use a link and these design tokens will then not be needed